dodola/tests/test_services.py

```diff
@@ -567,11 +567,11 @@ def test_aiqpd_train(tmpdir, monkeypatch):
     lon = [-99.83, -99.32, -99.79, -99.23]
     lat = [42.25, 42.21, 42.63, 42.59]
     time = pd.date_range(start="1994-12-17", end="2015-01-15")
```

Comment:
```suggestion
    time = xr.cftime_range(start="1994-12-17", end="2015-01-15", calendar="noleap")
```

Then I think you can remove your `import pandas as pd` and `from xclim.core.calendar import convert_calendar` lines above and the `ref_fine = convert_calendar(ref_fine["temperature"], target="noleap")` line below.

Reply:
That works!